src/ui/TopicExplorer.svelte

```diff
@@ -3,9 +3,33 @@
   import ONSAccordion from "./../ui/ons/ONSAccordion.svelte";
   import ONSAccordionPanel from "./../ui/ons/partials/ONSAccordionPanel.svelte";
   import censusData from "./../data/simpleTopicTableCategoryData";
+  import { onMount } from "svelte";
+  import slugify from "slugify";
+  import config from "../config";
 
-  let selectedData = getContext("selectedData");
+  export let selectedTopic;
+
+  let topicIndex;
+
+  $: {
+    if (selectedTopic) {
+      censusData.forEach((topic) => {
+        if (slugify(topic.name).toLowerCase() == selectedTopic.toLowerCase()) {
+          topicIndex = censusData.indexOf(topic);
+        }
+      });
+    }
+  }
+
+  onMount(() => {
+    if (selectedTopic) {
+      setTimeout(() => {
+        document.querySelector(`#topic-${topicIndex} .ons-btn`).click();
+      }, 150);
```

Comment:
This isn't sufficient for: https://deploy-preview-86--ons-data-map.netlify.app/categories/residency, I _believe_ Svelte has a native component somewhere to help with situations like this.

Reply:
I've increased the timeout to 250 and I also left a comment in the code to look for a different solution....